app/controllers/ideas_controller.rb

```diff
@@ -104,4 +106,9 @@ def idea_params
       )
     end
   end
+
+  def send_assigned_user_email
```

Comment:
Normally we try to not put stuff like this in controllers. It might be better to define it on `Idea`, `User`, or move it to its own class where we can call it directly from the controller action?

Here's also an [example](https://stackoverflow.com/questions/3861777/determine-what-attributes-were-changed-in-rails-after-save-callback) of a another way this could be done using callbacks on the model. So you could define an after_update callback on the idea model that sends the email if the assigned user changed.